### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#310] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181413](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/5e38a71f-b40c-49ed-85cb-ec76e19a4845)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/02be0631-497b-4d49-873a-54df120de10c)
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/81a8e784-9da2-4017-8fb4-df2813d01a58)
Nel punto 3° viene usata la proprietà dimostrata al punto 1:
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/60d5b072-8760-4c0c-ac5c-d847953b9bf2)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-20 182035](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/332e9a8d-655a-40c7-af3b-fa0005723911)
![Immagine 2024-01-20 182048](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b9e4d47c-bafc-4b31-a90b-fe2820f597fb)
![Immagine 2024-01-20 182109](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/a3b68e91-d806-4787-9016-a7db7ffbbf9f)
